openpype/hosts/resolve/hooks/pre_resolve_setup.py

```diff
@@ -41,61 +66,68 @@ def execute(self):
                 "/Applications/DaVinci Resolve/DaVinci Resolve.app"
                 "/Contents/Libraries/Fusion/fusionscript.so"
             ),
-            "linux": "/opt/resolve/libs/Fusion/fusionscript.so"
+            "linux": "/opt/resolve/libs/Fusion/fusionscript.so",
         }
-        RESOLVE_SCRIPT_LIB = os.path.normpath(
-            RESOLVE_SCRIPT_LIB_[current_platform])
-        self.launch_context.env["RESOLVE_SCRIPT_LIB"] = RESOLVE_SCRIPT_LIB
+        resolve_script_lib = Path(resolve_script_lib_dirs[current_platform])
+        self.launch_context.env[
+            "RESOLVE_SCRIPT_LIB"
+        ] = resolve_script_lib.as_posix()
+        self.log.info(
+            f"setting RESOLVE_SCRIPT_LIB variable to {resolve_script_lib}"
+        )
 
-        # TODO: add OTIO installation from  `openpype/requirements.py`
+        # TODO: add OTIO installation from `openpype/requirements.py`
         # making sure python <3.9.* is installed at provided path
-        python3_home = os.path.normpath(
-            self.launch_context.env.get("RESOLVE_PYTHON3_HOME", ""))
+        python3_home = Path(
+            self.launch_context.env.get("RESOLVE_PYTHON3_HOME", "")
+        )
 
-        assert os.path.isdir(python3_home), (
+        assert python3_home.is_dir(), (
             "Python 3 is not installed at the provided folder path. Either "
             "make sure the `environments\resolve.json` is having correctly "
             "set `RESOLVE_PYTHON3_HOME` or make sure Python 3 is installed "
             f"in given path. \nRESOLVE_PYTHON3_HOME: `{python3_home}`"
         )
-        self.launch_context.env["PYTHONHOME"] = python3_home
-        self.log.info(f"Path to Resolve Python folder: `{python3_home}`...")
-
-        # add to the python path to path
-        env_path = self.launch_context.env["PATH"]
-        self.launch_context.env["PATH"] = os.pathsep.join([
-            python3_home,
-            os.path.join(python3_home, "Scripts")
-        ] + env_path.split(os.pathsep))
-
-        self.log.debug(f"PATH: {self.launch_context.env['PATH']}")
+        python3_home_str = python3_home.as_posix()
+        self.launch_context.env["PYTHONHOME"] = python3_home_str
+        self.log.info(f"Path to Resolve Python folder: `{python3_home_str}`")
 
         # add to the PYTHONPATH
         env_pythonpath = self.launch_context.env["PYTHONPATH"]
-        self.launch_context.env["PYTHONPATH"] = os.pathsep.join([
-            os.path.join(python3_home, "Lib", "site-packages"),
-            os.path.join(RESOLVE_SCRIPT_API, "Modules"),
-        ] + env_pythonpath.split(os.pathsep))
+        modules_path = Path(resolve_script_api, "Modules").as_posix()
+        self.launch_context.env[
+            "PYTHONPATH"
+        ] = f"{modules_path}{os.pathsep}{env_pythonpath}"
 
         self.log.debug(f"PYTHONPATH: {self.launch_context.env['PYTHONPATH']}")
 
-        RESOLVE_UTILITY_SCRIPTS_DIR_ = {
+        # add to the python path to PATH
```

Comment:
```suggestion
        # add the pythonhome folder to PATH because on Windows
        # this is needed for Py3 to be correctly detected within Resolve
```